subset/brute/userList.json

```diff
@@ -0,0 +1,4 @@
+{
+  "username": "root,admin,manager",
```

Comment:
Use native json lists -- no need to encode them specifically here and then decode them in python.

Reply:
I still need to use a user_list as the next version will have ability to deal with multiple manufacturers

Reply:
What does that have to do with encoding this is a native json list?

Reply:
I want this to be a dynamic value? That I could potentially pass a different file to later on.
When you say native json you mean within the code ?

Reply:
fixed in user_list.json
fixed parsing in brute_client.py